code/modules/mob/living/silicon/ai/say.dm

```diff
@@ -135,6 +135,22 @@ var/const/VOX_PATH = "sound/vox_fem/"
 	for(var/word in words)
 		play_vox_word(word, src.z, null)
 
+	ai_voice_announcement_to_text(words, src)
+
+
+/proc/ai_voice_announcement_to_text(words, ai)

```

Comment:
Make this a proc on the AI level. It'll only be used by the AI anyway since it requires an AI as param

Reply:
good idea